pkg/domain/infra/abi/system.go

```diff
@@ -319,8 +319,8 @@ func (ic *ContainerEngine) SystemDf(ctx context.Context, options entities.System
 	}
 
 	dfVolumes := make([]*entities.SystemDfVolumeReport, 0, len(vols))
-	var reclaimableSize uint64
 	for _, v := range vols {
+		var reclaimableSize uint64
```

Comment:
Can we change line 344 to  `reclaimableSize = volSize`?
This will make it much less confusing to readers IMO.

Reply:
sure, I agree